libXfont 1.4 un-weak'ened this symbol (commit daa7af), resulting in:

/usr/lib/libXfont.a(fontfile.o):fontfile.c:(.text+0x0): multiple 
definition of `_CopyISOLatin1Lowered'
../../../dix/.libs/libdix.a(dixutils.o):dixutils.c:(.text+0xe0): first 
defined here

--- origsrc/xorg-server-1.6.0/dix/dixutils.c	2009-03-12 21:51:48.093750000 -0500
+++ src/xorg-server-1.6.0/dix/dixutils.c	2009-03-13 00:56:14.859375000 -0500
@@ -165,6 +165,7 @@ ISOLatin1ToLower (unsigned char source)
 }
 
 
+#ifndef __CYGWIN__
 _X_EXPORT void
 CopyISOLatin1Lowered(unsigned char *dest, unsigned char *source, int length)
 {
@@ -174,6 +175,7 @@ CopyISOLatin1Lowered(unsigned char *dest
 	*dest = ISOLatin1ToLower (*source);
     *dest = '\0';
 }
+#endif
 
 int
 CompareISOLatin1Lowered(unsigned char *s1, int s1len, 
